lib/forklift/box_distributor.rb

```diff
@@ -255,5 +248,17 @@ def symbolized_options(hash)
       hash.inject({}) { |memo, (k, v)| memo.update(k.to_sym => v) }
     end
 
+    def configure_forwarded_ports(box, override, bridged)
+      host_ip = box.fetch('forwarded_port_host_ip', '127.0.0.01')
+      if !bridged && box.fetch('name').to_s.include?('devel')
```

Comment:
I think a first line ```return if bridged``` will make it slightly easier to read. You can also move it outside of the if since you already have if bridged blocks there which can have an else block.